plugins/autowarmth.koplugin/main.lua

```diff
@@ -75,7 +75,7 @@ function AutoWarmth:onDispatcherRegisterActions()
     Dispatcher:registerAction("show_ephemeris",
         {category="none", event="ShowEphemeris", title=_("Show ephemeris"), general=true})
     Dispatcher:registerAction("auto_warmth_off",
-        {category="none", event="AutoWarmthOff", title=_("Auto warmth off"), screen=true})
+        {category="none", event="AutoWarmthOff", title=_("Auto warmth turned off"), screen=true})
```

Comment:
This change looks like a mistake? That is, in this context I would expect `Auto warmth off` to be short for `Turn off auto warmth`.